docs/modules/ROOT/pages/how-tos/creating.adoc

```diff
@@ -1 +1,56 @@
-= Creating applications and components
\ No newline at end of file
+= Creating applications and components
+
+To create an application in {ProductName}, someone must enable build pipelines in your instance of {ProductName}. This includes creating a GitHub application, so {ProductName} can access users' pull requests, and configuring a smee channel, to listen for those PRs. The first procedure in this document explains how to perform these tasks. 
+
+Once build pipelines are enabled, users can create their applications in {ProductName}. The second procedure below describes how to create a single-component application. For applications with multiple components...
+//Yftach: This is basically the question I had: What would users do to create an app with multiple components? Add a new URL to the same application-and-component.yaml? Create a new YAML file in the same namespace, within the Konflux directory?
```

Comment:
@yftacherzog: This is basically the question I had: What would users do to create an app with multiple components? Add a new URL to the same application-and-component.yaml? Create a new YAML file in the same namespace, within the Konflux directory?

Reply:
The number of yamls does not matter. It would be something like:

```yaml
---
apiVersion: appstudio.redhat.com/v1alpha1
kind: Application
metadata:
  name: test-application
  namespace: user-ns2
  annotations:
    application.thumbnail: "1"
spec:
  displayName: test-application
---
apiVersion: appstudio.redhat.com/v1alpha1
kind: Component
metadata:
  name: test-component1
  namespace: user-ns2
  annotations:
    build.appstudio.openshift.io/request: configure-pac
    image.redhat.com/generate: '{"visibility": "public"}'
spec:
  application: test-application
  componentName: test-component1
  source:
    git:
      revision: main
      url: https://github.com/konflux-ci/testrepo.git
      dockerfileUrl: Dockerfile
      context: ./
---
apiVersion: appstudio.redhat.com/v1alpha1
kind: Component
metadata:
  name: test-component2
  namespace: user-ns2
  annotations:
    build.appstudio.openshift.io/request: configure-pac
    image.redhat.com/generate: '{"visibility": "public"}'
spec:
  application: test-application
  componentName: test-component2
  source:
    git:
      revision: main
      url: https://github.com/konflux-ci/anothertestrepo.git
      dockerfileUrl: Dockerfile
      context: ./
---
apiVersion: "pipelinesascode.tekton.dev/v1alpha1"
kind: Repository
metadata:
  name: project-repository-testrepo
  namespace: user-ns2
spec:
  url: "https://github.com/konflux-ci/testrepo"
---
apiVersion: "pipelinesascode.tekton.dev/v1alpha1"
kind: Repository
metadata:
  name: project-repository-anothertestrepo
  namespace: user-ns2
spec:
  url: "https://github.com/konflux-ci/anothertestrepo"
```

Note that in some use cases, the same repository can be used for multiple components.

Reply:
Okkkk. Thank you. That's exactly what I was hoping to clarify. 

So I'll add this info--about how to include other components in this yaml--but otherwise, how does the doc look?